### ISSUE TITLE
document all options in man pages

### ISSUE THREAD

**Issue by pohly** [clearlinux/swupd-client#335]
--no-scripts was added recently to the code without also adding it to the swupd.1.rst. --picky has been undocumented even longer. That's just two options that I noticed, there's probably more.

**Comment by matthewrsj** [comment]
Total missing from man page (that I could find):
```
--no-scripts
--nosigcheck
--ignore-time
--certpath
--time
--no-boot-update
```

**Comment by matthewrsj** [comment]
Fixed by #367. If other out-of-date sections are identified this can be reopened or a new issue can be opened.
